ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -0,0 +1,68 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.numpy as ivy_np
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+
+
+# var
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy_np.valid_numeric_dtypes
+    ),
+    dtype=st.sampled_from(ivy_np.valid_numeric_dtypes + (None,)),
+    where=np_frontend_helpers.where(),
+    as_variable=helpers.array_bools(),
+    with_out=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.var"
+    ),
+    native_array=helpers.array_bools(),
+    container=st.booleans(),
+    instance_method=st.booleans(),
+    ddof=st.integers(),
+)
+def test_numpy_var(
+    *,
+    dtype_and_x,
+    dtype,
+    where,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    container,
+    instance_method,
+    ddof,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+    where = np_frontend_helpers.handle_where_and_array_bools(
+        where=where,
+        input_dtype=input_dtype,
+        as_variable=as_variable,
+        native_array=native_array,
+    )
+    np_frontend_helpers.test_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        container_flags=container,
+        instance_method=instance_method,
+        ddof=ddof,
+        fw=fw,
+        frontend="numpy",
+        fn_name="var",
+        x=np.asarray(x, dtype=input_dtype),
+        axis=None,
+        dtype=dtype,
+        out=None,
+        ddof=0,

```

Comment:
In test_averages_and_variances.py, Line 65, you have passed ddof a second time. You should remove this as you already pass ddof in Line 57.

Reply:
i fixed it it should be fine now